eli5/formatters/features.py

```diff
@@ -1,14 +1,16 @@
 # -*- coding: utf-8 -*-
 
+import six
+
 
 class FormattedFeatureName(object):
     """ Feature name that does not need any additional escaping.
     """
     def __init__(self, value):
         self.value = value
 
-    def __str__(self):
-        return str(self.value)
+    def format(self):
+        return six.text_type(self.value)
```

Comment:
What's six.text_type call for? When self.value can be bytes? In Python 2.x `unicode(b'string')` works correctly only for ascii-only byte strings because it tries to decode bytes data using sys.getdefaultencoding(). In Python3 str(b'foo') returns `"b'foo'"`, not `"foo"`.